bench/CMakeLists.txt

```diff
@@ -1,14 +1,25 @@
 aux_source_directory(. SRC_LIST)
+include_directories(..)
+include_directories(../utils)
 
-set(EXECUTABLE bench)
+add_executable(bench ${SRC_LIST})
 
-add_executable(${EXECUTABLE} ${SRC_LIST})
+find_package(Boost QUIET REQUIRED COMPONENTS chrono date_time)
```

Comment:
It seems that they are required only on Windows. How about moving them inside `if MSVC`?